.eslintrc

```diff
@@ -1,35 +1,122 @@
----
```

Comment:
https://github.com/postcss/postcss-plugin-boilerplate/blob/master/.eslintrc

Because it’s a PostCSS plugin and there is a PostCSS plugin boilerplate .eslintrc